src/modules/mavlink/mavlink_main.h

```diff
@@ -518,6 +518,8 @@ class Mavlink final : public ModuleParams
 
 	static hrt_abstime &get_first_start_time() { return _first_start_time; }
 
+	bool radio_status_critical() { return _radio_status_critical; }
```

Comment:
```suggestion
	bool radio_status_critical() const { return _radio_status_critical; }
```

Reply:
OK.  This one is good.  I'll try the commit suggestion feature here...